Source/WebCore/platform/graphics/cocoa/IOSurface.mm

```diff
@@ -268,34 +268,11 @@ static IntSize computeMaximumSurfaceSize()
     return size;
 }
 
-static WTF::Atomic<size_t>& surfaceBytesPerRowAlignment()
+size_t IOSurface::alignedBytesPerRow(size_t bytesPerRow)
 {
-    static WTF::Atomic<size_t> alignment = 0;
-    return alignment;
-}
-
-size_t IOSurface::bytesPerRowAlignment()
-{
-    auto alignment = surfaceBytesPerRowAlignment().load();
-    if (!alignment) {
-        surfaceBytesPerRowAlignment().store(IOSurfaceGetPropertyAlignment(kIOSurfaceBytesPerRow));
-        alignment = surfaceBytesPerRowAlignment().load();
-        // A return value for IOSurfaceGetPropertyAlignment(kIOSurfaceBytesPerRow) of 1 is invalid.
-        // See https://developer.apple.com/documentation/iosurface/1419453-iosurfacegetpropertyalignment?language=objc
-        // This likely means that the sandbox is blocking access to the IOSurface IOKit class,
-        // and that IOSurface::bytesPerRowAlignment() has been called before IOSurface::setBytesPerRowAlignment.
-        if (alignment <= 1) {
-            RELEASE_LOG_ERROR(Layers, "Sandbox does not allow IOSurface IOKit access.");
-            // 64 bytes is currently the alignment on all platforms.
-            alignment = 64;
-        }
-    }
-    return alignment;
-}
-
-void IOSurface::setBytesPerRowAlignment(size_t bytesPerRowAlignment)
-{
-    surfaceBytesPerRowAlignment().store(bytesPerRowAlignment);
+    constexpr size_t bytesPerRowAlignment = 64;
```

Comment:
previously IO surfaces were getting aligned via `IOSurfaceAlignProperty` which calls `IOSurfaceGetPropertyAlignment(kIOSurfaceBytesPerRow)` which can be and is often greater than 64 right? I tried on a J320 and it was 128. So this reduces the alignment across devices. Is that ok?